.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - "0.12"
   - "4"
   - "6"
-  - "stable"
+  - "7"
```

Comment:
Ah true, forgot that even though the odd numbered Node releases don't get LTS, they still overlap a bit with the even numbered releases.